tests/ui_tests/app/page/job/page.py

```diff
@@ -48,7 +48,9 @@ def __init__(self, driver, base_url, job_id: int):
 
     @allure.step("Check title on the page")
     def check_title(self, expected_title: str):
-        assert self.find_element(ObjectPageLocators.title).text == expected_title
+        self.wait_element_visible(ObjectPageLocators.title)
+        current_title = self.find_element(ObjectPageLocators.title).text
+        assert current_title == expected_title, f"Should be title '{expected_title}', but was {current_title}''"
```

Comment:
For further occasions: format of "noun verb" is easier to read, don't you think?
No need to actually change it, but "Title should be 'smt', not 'smt else'" looks better

Reply:
changed